lib/pubid/ieee/identifier.rb

```diff
@@ -11,11 +11,19 @@ def self.parse(code)
       new(**Parser.new.parse(code).to_h)
 
     rescue Parslet::ParseFailed => failure
-      raise Pubid::Iso::Errors::ParseError, "#{failure.message}\ncause: #{failure.parse_failure_cause.ascii_tree}"
+      raise Pubid::Ieee::Errors::ParseError, "#{failure.message}\ncause: #{failure.parse_failure_cause.ascii_tree}"
     end
 
     def to_s
-      "IEEE #{number}-#{year}"
+      "IEEE Std #{number}#{part}#{subpart}-#{year}"
```

Comment:
There are Standards vs non-Standards. Only Standards should have the "Std" label.